packages/components/src/markdown/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import omit from 'lodash/omit';
+import clsx from 'clsx';
+import marked from 'marked';
+import { invoke } from 'nebenan-helpers/lib/utils';
+import { stripOriginFromUrl } from 'nebenan-helpers/lib/routes';
+import { historyPropTypes } from 'nebenan-react-hocs/lib/history';
+import styles from './index.module.scss';
+
+import { sanitizeText } from './utils';
+
+const EMAIL_LINK_REGEX = /^mailto:/;
+const HTTP_LINK_REGEX = /^https?:/;
+const EXTENSION_REGEX = /\.\w+$/;
+
+const options = { gfm: false };
+
+class Markdown extends PureComponent {
```

Comment:
now is a good chance to refactor this in a functional way. won't need historyPropTypes/HOC anymore as well